apps/frontend/src/app/5_pages/LeaderboardPointsPage/components/ConnectWalletMessage/ConnectWalletMessage.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import { t } from 'i18next';
+
+import { Button } from '@sovryn/ui';
+
+import { useWalletConnect } from '../../../../../hooks';
+import { translations } from '../../../../../locales/i18n';
+
+export const ConnectWalletMessage = () => {
```

Comment:
Can you just reuse the existing component and move it to `app/1_atoms` or `app/2_molecules`?